### PR TITLE
Replace ws:// url scheme by parsec://

### PR DESCRIPTION
**Pull request by touilleMan**

### CODE REVIEW

**Comment by touilleMan** [comment]
related to #513

**Comment by touilleMan** [comment]
travis ci failure not related

File: parsec/types.py [hunk 1]
```diff
@@ -56,8 +56,8 @@ def __init__(self, raw: str):
 
         self._split = urlsplit(raw)
 
-        if self._split.scheme not in ("ws", "wss"):
-            raise ValueError("Backend addr must start with ws:// or wss://")
+        if self._split.scheme not in ("parsec", "parsecs"):
+            raise ValueError("Backend addr must start with parsec:// or parsecs://")
```
Comment (vxgmichel) [inline parsec/types.py]: Do we really need both `parsec` and `parsecs`?

Comment (touilleMan) [reply parsec/types.py]: `parsecs` for parsec-over-ssl mode, otherwise we could set the ssl with a url parameter

Comment (touilleMan) [reply parsec/types.py]: what do you think of 778006c ?

Comment (vxgmichel) [reply parsec/types.py]: much better! what's the point of supporting both ws and wss though?

Comment (touilleMan) [reply parsec/types.py]: ssl certificates are complex to set up (must ask a certificate from an authority or create our own and transfert it on each client machine) and can be handled by multiple components (for instance on heroku the ssl layer is directly handled by the infrastructure and our application receive clear data)
Hence it's simpler to consider ssl as an optional layer that can be easily setup according to the needs.

Comment (vxgmichel) [reply parsec/types.py]: If ssl is the default, shouldn't the option be `no_ssl`?

Comment (touilleMan) [reply parsec/types.py]: done.

**Comment by touilleMan** [comment]
Travis CI fail not related

**Pull request merged**